<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>654</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>654</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>654</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartCalc v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sub__button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mult__button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asin_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="div_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pow_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>79</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eq_button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>10</y>
      <width>441</width>
      <height>56</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod_button_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="XminText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X(min):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>590</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X(max):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y(min):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>590</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y(max):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="XmaxText">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>590</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="YminText">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>590</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="YmaxText">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>590</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_13">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button_14">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>14</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="XvalText">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>13</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mod_button_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
